src/components/Request/index.jsx

```diff
@@ -56,33 +60,38 @@ class RequestForm extends Component{
         error: true
       })
     })
+
+    const instructorEmail = `${this.state.selectedSlot.instructorId}@origincodeacademy.com`
+    const time = this.state.time;
+    const studentName = (this.props.user.firstName + ' ' + this.props.user.lastName);
+    axios.post(`/api/AptRequests/emailAdmin`, {
```

Comment:
This request should go inside the `.then` of the previous axios.post. We should expect that the request was successful before getting an email about it